tests/screenshot/apps/components/SliderButton.js

```diff
@@ -0,0 +1,9 @@
+import SliderButton from '../../../../SliderButton';
+import React from 'react';
+
+const SliderButtonTests = [
+	<SliderButton>{[]}</SliderButton>,
```

Comment:
I think the buttons should be displayed.
Currently,  only meaningless things are displayed.